hips/hip-0014.md

```diff
@@ -0,0 +1,84 @@
+---
+hip: 0014
```

Comment:
You need to quote the hip number or else it will interpreted as octal.

Reply:
oops - thanks